components/builder-api/src/server/resources/origins.rs

```diff
@@ -1221,7 +1372,7 @@ fn transfer_origin_ownership(req: HttpRequest,
 fn depart_from_origin(req: HttpRequest, path: Path<String>, state: Data<AppState>) -> HttpResponse {
     let origin = path.into_inner();
 
-    let session = match authorize_session(&req, None) {
+    let session = match authorize_session(&req, None, RoleAction::Read, None) {
```

Comment:
Should this require write + membership?

Reply:
Nope. In this case anyone should be able to leave an origin of their own volition.